NOAA-EMC/EVS
# Break up global_ens precip spatial plots job

## Description of Changes

This PR breaks the global_ens precip spatial plots job into two smaller jobs that run under 15 min, so restart capability is not needed. The first job will generate precip maps for gefs, cmce, and ecme. The second job will generate precip maps for naefs. This PR also reduces the warning messages to debug, so they won't appear in the log files.

## Developer Questions and Checklist
* Is this a high priorty PR? If so, why and is there a date it needs to be merged by? Yes.
* Do you have any planned upcoming annual leave/PTO? No.
* Are there any changes needed for when the jobs are supposed to run? One new job is added by this PR.
  
- [X] Jobs contain the approriate file checking and don't run METplus for any missing data.

## Testing Instructions
(1) Set up jobs

symlink the EVS_fix directory locally as "fix"

In the plots driver scripts, edit the following environment variables:

HOMEevs - set to your test EVS directory
COMIN - set to /lfs/h2/emc/vpppg/noscrub/emc.vpppg/${NET}_beta5/$evs_ver_2d

(2) Run jobs
Run the following jobs in EVS/dev/drivers/scripts/plots/global_ens:

qsub jevs_global_ens_atmos_gefs_precip_spatial_plots.sh
qsub jevs_global_ens_atmos_naefs_precip_spatial_plots.sh

A new evs.plots.global_ens.atmos.naefs.precip_spatial.vYYYYMMDD.tar file should appear containing NAEFS precip maps, and the evs.plots.global_ens.atmos.gefs.precip_spatial.vYYYYMMDD.tar file should only contains precip maps for GEFS, CMCE, and ECME.
